website/docs/cli/commands/test.mdx

```diff
@@ -1,13 +1,45 @@
 ---
 page_title: 'Command: test'
-description: Part of the ongoing design research for module integration testing.
+description: >-
+  The `terraform test` command loads and executes Terraform testing files.
 ---
 
 # Command: test
 
-The `terraform test` command is currently serving as part of
-[the module integration testing experiment](/terraform/language/modules/testing-experiment).
+The `terraform test` command reads in Terraform testing files and executes the tests detailed within.
 
-It's not ready for routine use, but if you'd be interested in trying the
-prototype functionality then we'd love to hear your feedback. See the
-experiment details page linked above for more information.
+The `test` command, and the test file syntax, are targeted at module authors wishing to validate and test their shared modules. It is also possible to use the `test` command to validate root modules.
```

Comment:
"targeted at" is slightly violent language, how would you feel about this instead:
```suggestion
The `test` command, and the test file syntax, are particularly helpful for module authors who want to validate and test their shared modules. You can also use the `test` command to validate root modules.
```